app/scripts/services/SMSInfo.svc.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+angular.module('yourStlCourts').factory('SMSInfo', function ($resource, $q) {
+  var SMSInfoResource = $resource('sms/info');
+  var smsInfo;
+
+  function formatPhoneNumber(numberToFormat){
```

Comment:
Take a look at making an angular filter for this:

http://jsfiddle.net/jorgecas99/S7aSj/